test/distributed/_sharded_tensor/ops/test_embedding.py

```diff
@@ -109,11 +132,13 @@ def test_sharded_embedding_colwise(self):
             self._run_sharded_embedding(spec, [4], 15, 14)
             self._run_sharded_embedding(spec, [34], 15, 14, padding_idx=10)
             self._run_sharded_embedding(spec, [8, 6, 5, 4], 23, 13, padding_idx=12)
-            self._run_sharded_embedding(spec, [4, 5, 6], 23, 13, max_norm=2.5)
```

Comment:
Why was this test removed?

Reply:
Opps, might be because of the merge conflict.